dev-libs/xxhash/xxhash-0.6.4.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A fast non-cryptographic hash algorithm"
+DESCRIPTION="xxHash is an extremely fast non-cryptographic hash algorithm, working at speeds close to RAM limits."
+HOMEPAGE="http://www.xxhash.com"
+COPYRIGHT="2012-2016 Yann Collet"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://github.com/Cyan4973/xxHash/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="4570ccd111df6b6386502791397906bf69b7371eb209af7d41debc2f074cdb22"
+SOURCE_DIR="xxHash-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="${portVersion%%.*}"
+
+PROVIDES="
+	libxxhash$secondaryArchSuffix = $portVersion
+	cmd:xxhsum = $portVersion
```

Comment:
```diff
-	cmd:xxhsum = $portVersion
+	cmd:xxhsum$secondaryArchSuffix = $portVersion
```

@extrowerk forgot to tell you that the `$secondaryArchSuffix` can only be dropped if we define a [`commandBinDir`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable).